vehicle-tracker/vehicle_tracker_app/lib/constants.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:latlong2/latlong.dart';
```

Comment:
Create an env file and move this to that env

Reply:
ok